setup.py

```diff
@@ -583,9 +583,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'croniter>=0.3.17, <0.4',
     'cryptography>=0.9.3',
     'dill>=0.2.2, <0.4',
-    'email_validator>=1.0.5, <2',   # Needed by WTForms (dependency of FAB)
     'flask>=1.1.0, <2.0',
-    'flask-appbuilder==2.3.2',
+    'flask-appbuilder==2.3.4',
```

Comment:
```suggestion
    'flask-appbuilder>=2.3.4',
```
WDYT?

Reply:
'flask-appbuilder>=2.3.4,<3'

Reply:
I am not 100% sure, but let's hope it won't break until 3

Reply:
Hope so, or to be on the safer side, we can do `flask-appbuilder~=2.3.4` i.e. >=2.3.4, == 2.3.*

Reply:
Sounds better imo